connector/src/main/java/com/google/cloud/spark/bigquery/ProtobufUtils.java

```diff
@@ -293,25 +343,32 @@ private static Object convertToProtoRowValue(
         || sparkType instanceof ShortType
         || sparkType instanceof IntegerType
         || sparkType instanceof LongType
-        || sparkType instanceof TimestampType
-        || sparkType instanceof DateType) {
```

Comment:
are we not going to support DateType now, or is it going to be encompassed in TimestampType

Reply:
So DateType is supported. It has just been moved since it's represented as a protobuf INT32, not an INT64 as all these other data-types.

TimeStamp's are fully supported on our side. I was waiting for Vortex to fix a bug with TimeStamp's, but it seems they might have fixed it. I am just awaiting confirmation, in order to support TimeStamp's on our side.

Reply:
I just checked with the BigQuery Storage Write API team and all the known issues with TimeStamp types have been fixed.